samza-kv/src/main/scala/org/apache/samza/storage/kv/KeyValueStorageEngine.scala

```diff
@@ -134,7 +147,11 @@ class KeyValueStorageEngine[K, V](
     info(count + " total entries restored.")
 
     if (batch.size > 0) {
-      rawStore.putAll(batch)
+      updateTimer(metrics.putAllNs) {
```

Comment:
There are multiple places where the putAll metrics are being updated in the same way. Could you consolidate into a "rawPutAll"? Or even have a doPutAll(keyValueStore, batch) which covers the regular putAll also.

Reply:
Sure, will do.